modules/monitoring-viewing-a-list-of-available-metrics.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-metrics.adoc
+
+:_content-type: PROCEDURE
+[id="viewing-a-list-of-available-metrics_{context}"]
+= Viewing a list of available metrics
+
+As a cluster administrator or as a user with view permissions for all projects, you can view a list of metrics available in a cluster and output the list in JSON format.
+
+.Prerequisites
+* You have installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-monitoring-view` role or have obtained a bearer token with `get` permission on the `namespaces` resource.
```

Comment:
It's better change to below:
You are cluster administrator or you have access to the cluster as a user with the `cluster-monitoring-view` role and have obtained thanos-querier  route .

reason:  `cluster-monitoring-view` user can not get 
cluster-admin grant cluster-monitoring-view clusterrole to user testuser-11
```
# oc adm policy add-cluster-role-to-user cluster-monitoring-view testuser-11
```
cluster-monitoring-view user login to the cluster
```
# oc login https://api.***.openshift.com:6443 -u testuser-11 -p ****
```
can not get thanos-querier  route 
```
# oc get routes -n openshift-monitoring thanos-querier -o jsonpath='{.status.ingress[0].host}'
Error from server (Forbidden): routes.route.openshift.io "thanos-querier" is forbidden: User "testuser-11" cannot get resource "routes" in API group "route.openshift.io" in the namespace "openshift-monitoring"
```

Reply:
I updated the content per @juzhao's suggestions.